includes/admin/extensions/class-extension-manager.php

```diff
@@ -0,0 +1,484 @@
+<?php
+
+namespace EDD\Admin\Extensions;
+
+use \EDD\Admin\Pass_Manager;
+use \EDD\Admin\Extensions\ExtensionsDownloadURL;
+
+class Extension_Manager {
+
+	/**
+	 * All of the installed plugins on the site.
+	 *
+	 * @since 2.11.x
+	 * @var array
+	 */
+	public $all_plugins;
+
+	/**
+	 * The minimum pass ID required to install the extension.
+	 *
+	 * @since 2.11.x
+	 * @var int
+	 */
+	private $required_pass_id;
+
+	/**
+	 * Pass Manager class
+	 *
+	 * @var \EDD\Admin\Pass_Manager
+	 */
+	protected $pass_manager;
+
+	public function __construct( $required_pass_id = null ) {
+		if ( $required_pass_id ) {
+			$this->required_pass_id = $required_pass_id;
+		}
+		$this->pass_manager = new Pass_Manager();
+
+		add_action( 'wp_ajax_edd_activate_extension', array( $this, 'activate' ) );
+		add_action( 'wp_ajax_edd_install_extension', array( $this, 'install' ) );
+		add_action( 'admin_enqueue_scripts', array( $this, 'register_scripts' ) );
+		// add_action( 'edd_plugin_activated', array( $this, 'delete_options' ) );
```

Comment:
I feel like I saw you address this at some point, but I can't find it. Can you refresh my memory as to why this is here (but also commented out)?